main.go

```diff
@@ -77,6 +77,7 @@ func main() {
 			Usage:       "Add local cluster (true, false)",
 			Value:       "true",
 			Destination: &config.AddLocal,
+			Hidden:      true,
```

Comment:
Can you remove this from the chart readme.md. Or update it to explain.

I believe @catherineluse already knows to update docs around helm chart, if not Catherine feel free to ping us.